modules/native_streaming_client_module/src/native_streaming_client_module_impl.cpp

```diff
@@ -575,7 +575,7 @@ StringPtr NativeStreamingClientModule::onCreateConnectionString(const ServerCapa
     }
 
     return NativeStreamingClientModule::createUrlConnectionString(
-        serverCapability.getProtocolId() == "opendaq_native_streaming" ? NativeStreamingPrefix : NativeConfigurationDevicePrefix,
```

Comment:
We need to be really careful on these since I'm pretty sure these break backwards compatibility. Anywhere on the client side where we check for protocol ID we need to make sure that we accept either the new or the old one.

So this concerns mainly:
- OPC UA client
- Native client
- Streaming LT client

Basically all of these should be checked for backwards compatibility. The easiest way to check is to start a simulator and connect to it *without* running `getAvailableDevices` before connecting, as this will use the capabilities of the device, not those created through discovery.